Dockerfile

```
@@ -44,3 +44,5 @@ HEALTHCHECK --interval=30s --timeout=5s --retries=3 CMD curl -f http://localhost
 
 # ✅ application-prod.yml을 명시적으로 설정에 포함
 ENTRYPOINT ["java", "-jar", "app.jar", "--spring.config.location=file:./application-prod.yml"]
+ENTRYPOINT ["java", "-jar", "app.jar", "--spring.config.location=classpath:/application.yml,classpath:/application-prod.yml,file:./application-prod.yml"]
```

Hadolint (2.12.0)

[error] Multiple `ENTRYPOINT` instructions found. If you list more than one `ENTRYPOINT` then only the last `ENTRYPOINT` will take effect

(DL4004)